resources/content/schema/admin/admin-auth.json

```diff
@@ -159,35 +235,46 @@
         "publish.transitioningProgress": "cr",
 
         "resourceDefinition" : "r",
+        "resourceDefinition.name" : "r",
+        
+        "registrationToken" : "cr",
+        "registrationToken.state":"r",
+        "registrationToken.registrationUrl" : "r",
+        
+        "register": "cr",
+        "register.key":"cr",
+        "register.secretKey" : "r",
+        "register.accessKey" : "r",
+        
+        "schema.*": "r",
+        
+        "statsAccess.*" : "r",
```

Comment:
I can see the case for wildcarding the properties on the schema resource (the previous line), but I don't see a compelling reason to wildcard statsAccess. No other resources' properties are wildcarded like